app/routes/member/invites/$invitationId/update.tsx

```diff
@@ -0,0 +1,5 @@
+import React from "react";
+
+export default function update() {
```

Comment:
this'll need to be named `action` for remix to know to call it when it receives a request